### ISSUE TITLE
Community Favourites Duplicate

### ISSUE THREAD

**Issue by xiningchen** [MemePlace/Frontend#38]
Not sure if it's just me, but currently the master branch contains code for the side navigation of communities. However when I favourite a community, it still shows up in the unfavourited with an empty yellow star and in the favourited... Since it shows up to the top of the community, I don't think we should have this duplication?

**Comment by Dave0921** [comment]
Yeah, I intended that to happen, just cause sometimes, the favourited communities won't be in the top 10. That said, I can change the functionality so that it removes the community from the unfavourited list.

**Comment by xiningchen** [comment]
Oh lol... wait I'm confused... If that's actually what we want, then don't change it. It was just strange that it duplicated...
